numba/npyufunc/workqueue.c

```diff
@@ -124,15 +128,16 @@ thread_pointer numba_new_thread(void *worker, void *arg)
     obj->arg = arg;
 
     handle = _beginthreadex(NULL, 0, bootstrap, obj, 0, &threadID);
-    if (handle == -1) return 0;
+    if (handle == -1)
+        return 0;
     return (thread_pointer)handle;
 }
 
-void take_a_nap(int usec) {
-    /* MS does not have usec resolution sleep function.
-    Sleeping for zero second will yield this thread.
-    */
-    Sleep(0);
+static void
+take_a_nap(int usec) {
+    /* Note that Sleep(0) will relinquish the current time slice, allowing
+       other threads to run. */
+    Sleep(usec / 1000);
```

Comment:
Is this the actual bug fix?  The rest of the PR seems like formatting fixes (which is fine, I'm just trying to understand where the problem was).